tests/models/test_match_spec.py

```diff
@@ -655,6 +655,30 @@ def test_parse_spec_str_with_brackets(self):
             "build": "3",
         }
 
+        # Ensure 'name' within brackets can't override the name specified outside of brackets
+        assert _parse_spec_str(
+            "tensorflow[name=* version=* md5=253b922ecdb5a30884875948b8904983]"
```

Comment:
this is a mistake, md5 should not be used for anything :)

Reply:
https://github.com/search?q=%22md5%22+filename%3Aenvironment.yml&type=Code

And yet it is out there!  I'm happy to add another test for sha256 or whatever the appropriate hash is